### DISCUSSION TITLE
Is actual watched time synced from Plex to Trakt?

### DISCUSSION THREAD

**Discussion by ztevo** [Taxel/PlexTraktSync#1886] [Q&A] [answered]
Hi
I have been using PTS for a year or so with one-way sync (Trakt to Plex), but am considering enabling 'watch status' with a two-way sync

I have two related questions please:

### **1) Can you kindly tell me if a watched time would be synced from Plex to Trakt? (And what time this would be?)**
_To be clear with an example:
-I finish watching an Episode on Plex at 5pm
-PTS is scheduled to run at 6pm, with a two-way sync
-PTS will copied the watched status from Plex to Trakt at 6pm._

_Will this show in Trakt as:
a) Episode being watched at 5pm (i.e. actual time/date the Episode was finished being watched)?
b) Episode being watched at 6pm (i.e. time/date the sync was completed)?
c) Something else?_

### **2) Could there be a situation where Plex would change the watched time of any items already marked as watched in Trakt?**

Thanks kindly in advance!
Steve

**Comment by glensc** [comment, marked as answer]
this application syncs watched time identically with "sync" command, two ways. do note that sync is unable to understand rewatches.

recording watched time with "watch" command is not that straight forward. as this app sends scrobble update, and once those are stopped, trakt deducts the watch time. either via api call that said stop, or it knew how long media is and when it should had ended.

sync command only makes change if there's no watch status on either side. and it will use timestamp from the other side. watch command on the other end will add new watch timestamp.

experiment and find answer yourself.

**Comment by ztevo** [comment]
Thank you that is helpful - was hesitant to test without knowing answer to second question (didn't want to change all the watched times!).

**Reply by glensc** [reply]
there's `--dry-run` flag for testing, it will not make any changes!
